package.json

```diff
@@ -306,7 +306,7 @@
     "json-source-map": "0.6.1",
     "jsurl": "^0.1.5",
     "lezer": "0.13.5",
```

Comment:
I think you will have to change these dependencies as well.

Basically `lezer` and `lezer-tree` can be replaced by `@lezer/lr` and `@lezer/common`. And since `@lezer/lr` depends on `@lezer/common` you could just replace `lezer` and `lezer-tree` dependencies by `@lezer/lr`.

You should end up with something like that:

```json
{
[...]
  "@lezer/lr": "^0.15.8"
[...]
  "jsurl": "^0.1.5",
  "lezer-promql": "0.22.0",
  "lodash": "4.17.21",
}
```

Reply:
yes, thank you 🙏